packages/next/src/client/page-loader.ts

```diff
@@ -64,7 +64,7 @@ export default class PageLoader {
         return window.__DEV_PAGES_MANIFEST.pages
       } else {
         this.promisedDevPagesManifest ||= fetch(
-          `${this.assetPrefix}/_next/static/development/${DEV_CLIENT_PAGES_MANIFEST}`
+          `${this.assetPrefix}/_next/static/development/${DEV_CLIENT_PAGES_MANIFEST}`,{credentials:'same-origin'}
```

Comment:
If we're applying this for the devPagesManifest, we should also apply it for the `devMiddlewareManifest` fetch and the stylesheets fetch as well 

x-ref: https://github.com/vercel/next.js/blob/e652a6f652c5f08ac87285e0f2140499103acd43/packages/next/src/client/page-loader.ts#L100-L102
x-ref: https://github.com/vercel/next.js/blob/e652a6f652c5f08ac87285e0f2140499103acd43/packages/next/src/client/route-loader.ts#L326

Reply:
ok @ijjk